fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -2293,10 +2354,79 @@ public Builder allowUniquePendingState(boolean allow) {
                 return this;
             }
 
+            /**
+             * Call {@link #allowTakeoverContinue(boolean)} (boolean)} with default true.
+             * @return this builder
+             */
+            public Builder allowTakeoverContinue() {
+                return this.allowTakeoverContinue(true);
+            }
+
+            /**
+             * In some special cases, an indexing method is allowed to continue building an index that was partly
+             * built by another method. If the other indexing session is still running, a "takeover" may cause it to fail.
+             * Note that it goes one way - once there is a "takeover", there is no way to continue building with the previous method.
+             * The current supported takeovers are:
+             *  * "Single index by records" may continue a multi index session.
+             *  * "Single index by records" may continue a mutually built session.
+             * @param allow if true, allow takeover.
+             * @return this builder
+             */
+            public Builder allowTakeoverContinue(boolean allow) {
+                this.doAllowTakeoverContinue = allow;
+                return this;
+            }
+
+            /**
+             * Call {@link #setMutualIndexing(boolean)} with default true.
+             * @return this builder
+             */
+            public Builder setMutualIndexing() {
```

Comment:
Perhaps worth tagging the mutual indexing code as `EXPERIMENTAL`

Reply:
✅